pallets/parachain-staking/src/tests.rs

```diff
@@ -9084,7 +9084,7 @@ fn test_on_initialize_weights() {
 
 			// add weight for invoking handle_delayed_payouts
 			// (goes away when we skip paying the first collator during round change)
-			let handle_delayed_payouts_weight = 536_374_000;
```

Comment:
@notlesh I hope this magic number is only temporarily here until we stop paying out rewards in first block, as this will probably always be updated from the expectation upon failure :(

Reply:
Yes, it should go away when we defer the first payout